sample/jobs/sampleProgrammaticBuild.groovy

```diff
@@ -0,0 +1,43 @@
+final String CONFIGURATION_REPO_URL = 'https://github.com/edx/configuration.git'
+// final String CONFIGURATION_BRANCH = 'master'
+final String CONFIGURATION_BRANCH = 'michael/travis' // temporary to be able to trigger builds
+
+final String EDX_REPO_ROOT = 'https://github.com/edx/'
+final String EDX_REPO_BRANCH = 'release'
+
+def job = job('configuration-watcher') {
+    // check out edx/configuration repository from GitHub
+    scm {
+        git {
+            remote {
+                url(CONFIGURATION_REPO_URL)
+                relativeTargetDir('configuration')
+                branch(CONFIGURATION_BRANCH)
+            }
+        }
+    }
+    // polls configuration repository for changes every 4 minutes
+    triggers {
+        scm('H/4 * * * *')
```

Comment:
This will be changed to H/10 * * * *; I was using 4 minutes so I could test more frequently.